.gitignore

```diff
@@ -8,3 +8,5 @@
 cypress/fixtures/example.json
 cypress/videos
 cypress/screenshots
+
+.vscode/settings.json
```

Comment:
You could add this to your local, global gitignore instead, see: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/